layout/identifier.go

```diff
@@ -0,0 +1,28 @@
+package layout
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+)
+
+type IDIdentifier struct {
+	ImageID string
+}
+
+func newIDIdentifier(configFile *v1.ConfigFile) (IDIdentifier, error) {
+	return IDIdentifier{
+		ImageID: "sha256:" + asSha256(configFile),
```

Comment:
I think elsewhere we call this "digest"

Reply:
Yeah, I took the same name used in remote package